server/services/courseService.test.ts

```diff
@@ -19,6 +20,46 @@ describe('CourseService', () => {
     courseClientBuilder.mockReturnValue(courseClient)
   })
 
+  describe('createParticipation', () => {
+    const person = personFactory.build()
+
+    it('creates a course participation', async () => {
+      const courseId = faker.string.uuid()
+      const otherCourseName = ''
+      const courseParticipation = courseParticipationFactory.withCourseId().build()
+
+      when(courseClient.createParticipation)
+        .calledWith(person.prisonNumber, courseId, undefined)
+        .mockResolvedValue(courseParticipation)
+
+      const result = await service.createParticipation(token, person.prisonNumber, courseId, otherCourseName)
+
+      expect(result).toEqual(courseParticipation)
+
+      expect(courseClientBuilder).toHaveBeenCalledWith(token)
+      expect(courseClient.createParticipation).toHaveBeenCalledWith(person.prisonNumber, courseId, undefined)
+    })
+
+    describe('when the courseId is "other"', () => {
+      it('creates a course participation with the other course name', async () => {
+        const courseId = 'other'
+        const otherCourseName = 'Other course'
+        const courseParticipation = courseParticipationFactory.withOtherCourseName().build()
```

Comment:
It doesn't matter because of the mocking, but I wonder if it's worth passing the `otherCourseName` into build here for tests as documentation purposes? Likewise with `courseId` in the previous test